### PR TITLE
Add announcement banner for image registry freeze

### PR DESCRIPTION
**Pull request by upodroid**
/cc @BenTheElder @dims 

KEP: https://github.com/kubernetes/enhancements/issues/3720

### CODE REVIEW

**Comment by upodroid** [comment]
What colour should the banner be? I think red is a bit loud.

**Comment by sftim** [comment]
/committee steering

Needs steering committee approval

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -130,3 +130,13 @@ announcements:
   message: |
     5 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     October 24 - 28, 2022.
+
+- name: Freezing k8s.gcr.io
+  startTime: 2023-02-21T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
+                                 # This should run before and after Kubecon EU 2023
+  endTime: 2023-04-15T00:00:00
```
Comment (sftim) [inline data/announcements/scheduled.yaml:137]: Watch out for grammar. Ideally, we use a different wording from the 3rd of April.

File: data/announcements/scheduled.yaml [hunk 2]
```diff
@@ -130,3 +130,13 @@ announcements:
...
+  style: "background: #FF0000"
+  title: "k8s.gcr.io Image Registry Will Be Frozen From April 2023"
```
Comment (sftim) [inline data/announcements/scheduled.yaml:139]:
```suggestion
  title: "k8s.gcr.io container image registry will be frozen in early April 2023"
```

File: data/announcements/scheduled.yaml [hunk 3]
```diff
@@ -130,3 +130,13 @@ announcements:
   message: |
     5 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     October 24 - 28, 2022.
+
+- name: Freezing k8s.gcr.io
+  startTime: 2023-02-21T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
+                                 # This should run before and after Kubecon EU 2023
+  endTime: 2023-04-15T00:00:00
+  style: "background: #FF0000"
```
Comment (sftim) [inline data/announcements/scheduled.yaml:138]: A darker shade of red will help with accessibility, and also doesn't look so much like a “we're shutting down Kubernetes” message.

File: data/announcements/scheduled.yaml [hunk 4]
```diff
@@ -130,3 +130,13 @@ announcements:
...
+  title: "k8s.gcr.io Image Registry Will Be Frozen From April 2023"
+  message: |
+    k8s.gcr.io image registry will be frozen from the 3rd of April 2023.<br/>
```
Comment (sftim) [inline data/announcements/scheduled.yaml:141]:
```suggestion
    The legacy `k8s.gcr.io` image registry will be frozen from the 3rd of April 2023.<br/>
```

**Review by sftim — COMMENTED** [review]
My thoughts; hope these help.

File: data/announcements/scheduled.yaml (hunk 1 above)
Comment (sftim) [reply data/announcements/scheduled.yaml:137]: eg: “Legacy k8s.gcr.io container image registry was frozen on the 3rd of April 2023”

**Comment by upodroid** [comment]
image

File: data/announcements/scheduled.yaml [hunk 5]
```diff
@@ -130,3 +130,25 @@ announcements:
   message: |
     5 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     October 24 - 28, 2022.
+
+- name: Freezing k8s.gcr.io - Before
+  startTime: 2023-02-20T00:00:00 # Added in https://github.com/kubernetes/website/pull/39575
+                                 # This should run before and after Kubecon EU 2023
+  endTime: 2023-04-02T00:00:00
```
Comment (mrbobbytables) [inline data/announcements/scheduled.yaml:137]: there's another PR open to update this with another announcement from 3/27 to 4/4. I'd change it to end on 3/26 and we can put it back on 4/5 to run through kubecon eu

Comment (upodroid) [reply data/announcements/scheduled.yaml]: Fixed

Comment (mrbobbytables) [reply data/announcements/scheduled.yaml]: thanks!

**Comment by mrbobbytables** [comment]
lgtm with steering hat on

we will probably need to rebase this or the other banner PR based on whichever merges first

**Comment by tpepper** [comment]
+1 with Steering hat on.  If this and https://github.com/kubernetes/website/pull/39257 both go in, can we make this one stand out more?

**Comment by sftim** [comment]
We can only have one banner up at a time and there's a precedence order (based on which appears first in the YAML).

I don't mind rebasing my PR, I know how the mechanisms all work as I helped make 'em.

**Comment by sftim** [comment]
I still think the background is too bright mind. For accessibility we should darken it.

**Comment by sftim** [comment]
LGTM for docs
@tengqm - if you're around - does this look OK to you?

**Comment by upodroid** [comment]
/label tide/merge-method-squash

I used a darker shade of red.

**Comment by upodroid** [comment]
image

**Comment by sftim** [comment]
/lgtm

Ready for formal approval by steering committee

**Comment by sftim** [comment]
BTW if you keep the same [merge base](https://git-scm.com/docs/git-merge-base) you can squash this to 1 commit in your fork @upodroid; that's a bit tidier. And Prow leaves the LGTM in place.

**Comment by cblecker** [comment]
/approve
/hold

Hold for other comments from steering.

File: data/announcements/OWNERS [hunk 6]
```diff
@@ -6,3 +6,6 @@ options:
   no_parent_owners: true
 approvers:
 - committee-steering # defined in OWNERS_ALIASES
+
+labels:
+- committee/steering
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply data/announcements/OWNERS:11]: I agree that this change should go as a separate commit. But, this PR has the label `tide/merge-method-squash`. Even if there are separate commits, the history would have only one commit.

I suggest pruning the commit history in the PR to the following commits:
1. Banner changes
2. Change to label

And then remove the label to squash.

**Comment by sftim** [comment]
/hold

for https://github.com/kubernetes/website/pull/39575#discussion_r1116818112

File: data/announcements/OWNERS (hunk 6 above)
[the remark this answers is not in the document]
Comment (upodroid) [reply data/announcements/OWNERS:11]: I'll add the labels in a separate PR.

Fixed.

**Comment by sftim** [comment]
@upodroid what's your intention here; are you willing to squash this in your fork? (that's ideal)

**Comment by upodroid** [comment]
We squash merge this PR and I open a new one to fix the labels.

**Comment by upodroid** [comment]
We squash merge this PR and I open a new one to fix the labels.

**Review by palnabarun — COMMENTED** [review]
/approve
(with the steering hat)

**Comment by palnabarun** [comment]
Tracking approval from Steering Committee:
- [ ] @BenTheElder 
- [ ] @cblecker 
- [x] @cpanato 
- [ ] @justaugustus 
- [ ] @mrbobbytables 
- [x] @palnabarun 
- [ ] @tpepper

**Review by BenTheElder — COMMENTED** [review]
Sorry I've been unavailable. Thank you!
/lgtm
/approve

**Comment by sftim** [comment]
Sorry @upodroid - can you rebase this? We've made a breaking change to the CVE feed since you opened the PR.

PS You might want to revise the start date too given the current date. That's optional though.

**Comment by sftim** [comment]
I think this is good to unhold given approvals once it has a new LGTM from SIG Docs, and we've checked that any rebase wasn't malicious.

**Comment by cblecker** [comment]
@sftim Agreed, you can go ahead from the steering side :)

**Comment by upodroid** [comment]
Fixed

/hold cancel

**Comment by sftim** [comment]
You are very welcome to suggest a feature request for dismissable banners @AntonioND - see https://github.com/kubernetes/website/issues/new/choose

Right now, we need to put up a message, and we don't have dismissable banners. Help is welcome.

**Pull request merged**